compiler/rustc_middle/src/query/mod.rs

```diff
@@ -147,7 +147,7 @@ rustc_queries! {
     /// we will have to assume that any change means that you need to be recompiled.
     /// This is because the `hir_crate` query gives you access to all other items.
     /// To avoid this fate, do not call `tcx.hir().krate()`; instead,
-    /// prefer wrappers like `tcx.visit_all_items_in_krate()`.
+    /// prefer wrappers like `tcx.hir_visit_all_item_likes_in_crate()`.
```

Comment:
maybe use a rustdoc link instead so that this doesn't regress anymore if the function is renamed again (and it'll improve exploring this function's documentation)

Reply:
Good idea. Done.